src/service/trigger.ts

```
@@ -321,7 +395,149 @@ function createStoredSession(bot: Bot, temp: WakeUpReplyRecord) {
+        if (!next) return null
+        if (next.getTime() > baseAt) return next.getTime()
+        next.setDate(next.getDate() + 1)
+        return next.getTime()
+    }
+
+    const weekly = raw.match(/^([1-7])-(\d{2}):(\d{2}):(\d{2})$/)
+    if (repeatRule === 'weekly' && weekly) {
+        const weekday = Number.parseInt(weekly[1], 10) % 7
+        const next = buildDate(year, month, day, weekly[2], weekly[3], weekly[4])
+        if (!next) return null
+        next.setDate(next.getDate() + (weekday - next.getDay() + 7) % 7)
```

ESLint

[error] Replace `weekday·-·next.getDay()·+·7)·%·7` with `(weekday·-·next.getDay()·+·7)·%·7)`

(prettier/prettier)